Cargo.toml

```diff
@@ -40,6 +40,7 @@ serde_json = "~1.0.8"
 [features]
 mock = ["lru_time_cache/fake_clock", "safe_crypto/mock", "parsec/mock", "parsec/malice-detection"]
 mock_parsec = ["mock"]
+mock_serialise_send = ["mock_parsec"]
```

Comment:
we prolly also want to update https://github.com/maidsafe/routing/blob/fleming/scripts/tests#L7-L8 with this PR

Reply:
We can, I was not sure if we wanted to change the CI.

Option 1: keep it as is (not so fast not so slow, test serialization)
Option 2: test both (slowest option: need to build twice and run twice).
Option 3: only test with no serialization: then missing serialization coverage.

That is why I left it as is as I thought it may be only used for soak testing initially.

Reply:
hmm its a fair point, sure we can stick to option-1 I guess for now until CI time becomes an issue in this repo or due to this repo for other shared runners. Ideally yeh would be good for this to be in opt-3 and go with opt-2 for something like nightly/weekly CI